src/uikit.cs

```diff
@@ -7939,6 +8267,18 @@ interface UIImage : NSSecureCoding
 		[Static]
 		[Export ("strokedCheckmarkImage", ArgumentSemantic.Strong)]
 		UIImage StrokedCheckmarkImage { get; }
+
+		[Watch (9,0), TV (16,0), MacCatalyst (16,0), iOS (16,0)]
+		[Static]
+		[Export ("systemImageNamed:variableValue:withConfiguration:")]
+		[return: NullAllowed]
+		UIImage GetSystemImage (string name, double value, [NullAllowed] UIImageConfiguration configuration);
+
+		[Watch (9,0), TV (16,0), MacCatalyst (16,0), iOS (16,0)]
+		[Static]
+		[Export ("imageNamed:inBundle:variableValue:withConfiguration:")]
+		[return: NullAllowed]
+		UIImage GetImageNamed (string name, [NullAllowed] NSBundle bundle, double value, [NullAllowed] UIImageConfiguration configuration);
```

Comment:
We have other `imageNamed:...` overloads named `FromBundle`:

```suggestion
		UIImage FromBundle (string name, [NullAllowed] NSBundle bundle, double value, [NullAllowed] UIImageConfiguration configuration);
```